### ISSUE TITLE
Any idea on how to fix rubberbanding ?

### ISSUE THREAD

**Issue by MoryCorp** [VeroFess/PalWorld-Server-Unoffical-Fix#33]
The patch did help for the memory leaks but I'm still experiencing a lot of rubber bands on a 32Gb 12 cores server

**Comment by VeroFess** [comment]
Yes, I have found where the problem occurs so far, but the patch requires a lot of development and testing, so it won't be rolled out very quickly.

**Comment by MoryCorp** [comment]
Thanks for the answer !  
Have you been able to find how to edit the 30sec autosave as it could also helps the lags ?

**Comment by raikxou** [comment]
hihi sorry to rebump this but could you mind sharing where the problem occurs that you found?
